### PR TITLE
Updates to Docker EE instructions

### PR DESCRIPTION
**Pull request by rainest**
Updating Docker EE instructions with the following:
* RBAC instructions include user creation steps and link to larger RBAC article
* Separate Vitals instructions removed (the initial run already enables it)
* Moved FAQ to end of article. Fixed issues with formatting
* Instructions for trial users

### CODE REVIEW

File: app/docs/enterprise/0.31-x/installation/docker.md [hunk 1]
```diff
@@ -74,33 +80,37 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
...
-## Enable RBAC and Vitals
+## FAQs
 
-Finally, two of the key features of 0.29+ are RBAC and Vitals. To enable them:
+Starting with Kong `0.30`, the Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk with the Kong Proxy.
 
-1. Start a bash session on the container
-        
-        docker exec -it kong-ee /bin/sh
+Starting with 0.29, without a license properly referenced, you’ll get errors running migrations. Also, without a license, you'll do a “docker start <name>” and not see an error attempting to start the container. But when you check the process, it won’t be running. Doing a “docker logs <container_name>” will show you:
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/installation/docker.md]: I'm thinking that messages about versions as old as this are no longer needed.

File: app/docs/enterprise/0.31-x/installation/docker.md [hunk 2]
```diff
@@ -74,33 +80,37 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
...
+        curl -X GET http://localhost:8001/status -H "Kong-Admin-Token: 12345"
 
-    Your license data must contain only straight quotes to be considered valid JSON.
+If you are able to access Kong without issues, you can add `KONG_ENFORCE_RBAC=on` to your initial container environment variables.
 
-## Enable RBAC and Vitals
+## FAQs
 
-Finally, two of the key features of 0.29+ are RBAC and Vitals. To enable them:
+Starting with Kong `0.30`, the Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk with the Kong Proxy.
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/installation/docker.md]: Same here regarding older versions - I think you can take out the "Starting with Kong 0.30" part.

File: app/docs/enterprise/0.31-x/installation/docker.md [hunk 3]
```diff
@@ -74,33 +80,37 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
...
-## Enable RBAC and Vitals
+## FAQs
 
-Finally, two of the key features of 0.29+ are RBAC and Vitals. To enable them:
+The Admin API only listens on the local interface by default. This was done as a security enhancement. Note that here, we are overriding that in the above example with `KONG_ADMIN_LISTEN=0.0.0.0:8001` because Docker container networking benefits from more open settings and enables the Admin GUI & Dev Portal to talk with the Kong Proxy.
 
-1. Start a bash session on the container
-        
-        docker exec -it kong-ee /bin/sh
+Without a license properly referenced, you’ll get errors running migrations. Also, without a license, you'll do a “docker start <name>” and not see an error attempting to start the container. But when you check the process, it won’t be running. Doing a “docker logs <container_name>” will show you:
```
Comment (nijikokun) [inline app/docs/enterprise/0.31-x/installation/docker.md]: The docker commands here should be wrapped in backticks to be consistent with other commands and variables throughout the documentation.

File: app/docs/enterprise/0.31-x/installation/docker.md [hunk 4]
```diff
@@ -74,33 +80,37 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
...
 
-        nginx: [alert] Error validating Kong license: could not decode license json
+2. Start a bash session on the container
+        
+        docker exec -it kong-ee /bin/sh
+
+3. KONG_ENFORCE_RBAC=on kong reload
+4. Confirm that your user token is working by passing the `Kong-Admin-Token` header in requests
+        
+        curl -X GET http://localhost:8001/status -H "Kong-Admin-Token: 12345"
```
Comment (nijikokun) [inline app/docs/enterprise/0.31-x/installation/docker.md]: Small nitpick, instructions should end with colons to stay consistent with the rest of the document.

Comment (thibaultcha) [reply app/docs/enterprise/0.31-x/installation/docker.md]: True! They should also start with `$` as all other CLI instructions as well

**Pull request merged**